include/odp/api/spec/ml_types.h

```diff
@@ -329,6 +329,13 @@ typedef struct odp_ml_capability_t {
 
 /** Machine learning configuration parameters */
 typedef struct odp_ml_config_t {
+	/** Engine ID to be configured.
+	 *
+	 * In a system with multiple ML engines, this parameter selects the engine to be configured.
+	 * The default value is 0.
+	 */
+	uint32_t engine_id;
```

Comment:
I am not sure if we should use the term 'engine' or 'device'. Engine would make sense if we think that there, at some point, could be multiple devices, each of which could contain multiple engines. But I suppose we are not preparing for that. 'Engine' sounds ok to me, but I wonder if the same concept in other API's is more likely to be called 'device' and therefore 'device' would be better here. Maybe it does not matter.

Reply:
Based on discussions, we would like to go ahead with 'engine'.